cli/stream_command.go

```diff
@@ -1863,12 +1867,36 @@ func (c *streamCmd) showStreamInfo(info *api.StreamInfo) {
 
 		if s.SubjectTransformDest != "" {
 			filter := ">"
+
 			if s.FilterSubject != "" {
 				filter = s.FilterSubject
 			}
-			cols.AddRowf("Subject Transform", "%s to %s", filter, s.SubjectTransformDest)
+
+			cols.AddRowf("Subject Filter and Transform", "%s to %s", filter, s.SubjectTransformDest)
 		} else {
-			cols.AddRowIfNotEmpty("Subject Filter", s.FilterSubject)
+			if s.SubjectTransformDest == "" && len(s.SubjectTransforms) == 0 {
+				cols.AddRowIfNotEmpty("Subject Filter", s.FilterSubject)
+			} else {
+				for i := range s.SubjectTransforms {
+					var t string
+
+					if i == 0 {
+						if len(s.SubjectTransforms) > 1 {
+							t = "Subject Filters and Transforms"
+						} else {
+							t = "Subject Filter and Transform"
+						}
+					} else {
```

Comment:
else clause not needed `var t string` makes an empty string, this is why I prefer `t := ""` instead when making this kind of var because it becomes much more obvious then

Reply:
Done